### PR TITLE
No 19

### PR DESCRIPTION
**Pull request by dmbellinger**

### CODE REVIEW

**Comment by dmbellinger** [comment]
intakeDelete
intakeGET
intakePOST
intakePUT

**Comment by kevin-rodri** [comment]
You have merge conflicts to resolve. Also, thank you for testing the endpoints you created.

File: associations.js [hunk 1]
```diff
@@ -20,6 +20,10 @@ function setupAssociations(sequelize) {
     // 1:N relationship between Patient and PatientOrder
     Patient.hasMany(PatientOrders, { foreignKey: "patient_id" });
     PatientOrders.belongsTo(Patient, { foreignKey: "patient_id" });
+
```
Comment (kevin-rodri) [inline associations.js]: :+ 1 for this change!

**Review by kevin-rodri — COMMENTED** [review]
Code quality is improving from the last few PRs you've had up. The only thing that's left is to fix the merge conflict you have, and then you can start the next task. Nice work.

File: associations.js
Comment (kevin-rodri) [inline associations.js:1]: You're missing your changes.

File: associations.js [hunk 2]
```diff
@@ -7,23 +7,35 @@ Per: https://github.com/sequelize/express-example/blob/master/express-main-examp
...
-  
-    // 1:N relationship between Patient and PatientOrder
-    Patient.hasMany(PatientOrders, { foreignKey: "patient_id" });
-    PatientOrders.belongsTo(Patient, { foreignKey: "patient_id" });
-
-    // 1:N relationship between Intake and Patient
-    Patient.hasMany(Intake, { foreignKey: "patient_id" });
-    Intake.belongsTo(Patient, { foreignKey: "patient_id" });
+
+  const {
```
Comment (kevin-rodri) [inline associations.js:11]: Don't forget about the Intake model.

File: associations.js [hunk 3]
```diff
@@ -7,23 +7,35 @@ Per: https://github.com/sequelize/express-example/blob/master/express-main-examp
...
+  PatientOrders.belongsTo(Patient, { foreignKey: "patient_id" });
+
+  // 1:N relationship between Patient and VitalSigns
+  Patient.hasMany(VitalSigns, { foreignKey: "patient_id" });
+  VitalSigns.belongsTo(Patient, { foreignKey: "patient_id" });
+
+  // 1:1 relationship where VitalSigns has one PatientPainScale
+  VitalSigns.hasOne(PatientPainScale, { foreignKey: "id" });
+  PatientPainScale.belongsTo(VitalSigns, { foreignKey: "id" });
 }
```
Comment (kevin-rodri) [inline associations.js:45]: You also need to add this: 

`// 1:N relationship between Intake and Patient
     Patient.hasMany(Intake, { foreignKey: "patient_id" });
     Intake.belongsTo(Patient, { foreignKey: "patient_id" });`

**Review by kevin-rodri — CHANGES_REQUESTED** [review]

**Review by kevin-rodri — APPROVED** [review]
**Approved!**

**Pull request merged**